index.html

```diff
@@ -73,7 +73,9 @@
     { name: "Manu Sporny", url: "http://digitalbazaar.com/",
       company: "Digital Bazaar, Inc.", companyURL: "http://digitalbazaar.com/" },
     { name: "Ken Ebert", url: "",
-      company: "Sovrin Foundation", companyURL: "https://sovrin.org/" }
+      company: "Sovrin Foundation", companyURL: "https://sovrin.org/" },
+    { name: "David W Chadwick", url: "https://www.linkedin.com/in/davidwchadwick/",
+      company: "Truetrust Ltd", companyURL: "https://truetrust.co.uk/" },
```

Comment:
```suggestion
```

We'll update this later. For now, add a credit line as I'll demonstrate in another comment.